apps/v4/content/docs/components/masonry.md

```
@@ -0,0 +1,112 @@
+
+::tabs-content{value="cli"}
+
+```bash
+npx shadcn-vue@latest add masonry
+```
+
+::
+
+::tabs-content{value="manual"}
+  ::steps
+    ::step
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)